bin/pa11y.js

```diff
@@ -9,6 +9,18 @@ const program = require('commander');
 const pa11y = require('../lib/pa11y');
 const semver = require('semver');
 
```

Comment:
should we create an issue in the pa11y repo for this and link to it here? can't decide 😁 i feel like maybe we'll forget if we can't see it in the issues list

Reply:
Good idea 👍  will in a sec if you haven't (haven't checked :D)